config.toml

```diff
@@ -4,29 +4,67 @@ title = "Kiali"
 theme = "canvas"
 
 [params]
-    custom_css = ["/css/kiali_style.css"]
+    custom_css = [
+      "/css/kiali_style.css",
+      "https://use.fontawesome.com/releases/v5.3.1/css/all.css"
+    ]
 
 [menu]
   [[menu.main]]
-  identifier = "swagger"
-  name = "API"
-  url = "/api/overview"
-  weight = 40
+      identifier = "swagger"
```

Comment:
why are all the diffs in this file needed? some are only tabs, some are comment out of part of the content (they might be, I just want to make sure)

Reply:
Sorry I resolved the review :'(  Additional css for asccidocs icons, tabulation format to be more readable and I added the full documentation with comments to test the Index for documentation and let to the next editors the menu ready.